### ISSUE TITLE
New training dataset with saved model

### ISSUE THREAD

**Issue by qazs** [lvapeab/nmt-keras#73]
Hi,
Would it affect the accuracy if I were to use a new training dataset with a saved model?
Let's say I have trained and saved a model with training dataset A.
Can I use a new training dataset B to resume training? Or should I always append new training data to dataset A and retrain everything?

**Comment by lvapeab** [comment]
The accuracy depends on how related are the domains A and B. If they are close, the performance will probably be high. If they are very different, the performance will likely decrease.

You can continue a training but changing the data. You should set the `config.py` for loading a model, set `RELOAD_DATASET` to `False` and change the `TEXT_FILES` (and/or optionally the `DATA_ROOT_PATH`) to point the new training data.

**Comment by qazs** [comment]
G

**Comment by lvapeab** [comment]
It may be a regular behavior. As the new domain is unknown by the model, it may cause the model to get "confused", increasing its loss. 

However, this can also mask an irregular behaviour. You can test it by resuming the training with the same domain/dataset and observe whether there are these loss fluctuations.

**Comment by qazs** [comment]
I was actually resuming training with the same dataset and configurations..
